### DISCUSSION TITLE
Support sending bits (cheering)

### DISCUSSION THREAD

**Discussion by CaptainFlint** [Chatterino/chatterino2#3867] [Ideas] [answered]
When I tried sending some bits, they appeared in chat as normal text ("cheerNNN"), without actually sending bits. When I type the same message in the webchat, it activates the cheering mode and sends the specified amount of bits with (or without) a message. It would be nice if we could send cheers directly from Chatterino the same way, as from the webchat, even without any interactivity.

**Comment by Mm2PL** [comment, marked as answer]
This is impossible for us. We rely on Twitch's public API which doesn't have such functionality. As much as it would be nice, it has very high potential for abuse, for example someone stealing your Chatterino authentication token and then taking all your bits and maybe auto-purchasing some.

**Comment by CaptainFlint** [comment]
I see. Thank you for replying, I didn't know it was missing from the public API.
